### PR TITLE
feat: bench Erc721 gas usage

### LINKED ISSUE #171: Gas benches for ERC-721
Add benches based on the work in #168

### PR DESCRIPTION
**Pull request by alexfertel**
Resolves #171

### CODE REVIEW

File: benches/src/main.rs [hunk 1]
```diff
@@ -1,9 +1,10 @@
-use benches::{access_control, erc20, merkle_proofs};
+use benches::{access_control, erc20, erc721, merkle_proofs};
 
 #[tokio::main]
 async fn main() -> eyre::Result<()> {
     let _ = tokio::join!(
         erc20::bench(),
+        erc721::bench(),
```
Comment (bidzyyys) [inline benches/src/main.rs:7]: Nit: I would apply alphabetic order

File: benches/src/erc721.rs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        .wallet(EthereumWallet::from(alice.signer.clone()))
+        .on_http(alice.url().parse()?);
+
+    let bob = Account::new().await?;
+    let bob_addr = bob.address();
+
+    let contract_addr = deploy(&alice).await;
+    let contract = Erc721::new(contract_addr, &alice_wallet);
+
+    let _ = receipt!(contract.mint(alice_addr, uint!(2_U256)))?;
```
Comment (bidzyyys) [inline benches/src/erc721.rs:47]: Nit: I would create `let token_2 = uint!(2_U256);` and then reuse it.

**Review by bidzyyys — APPROVED** [review]
Looks good 🚀

**Review by qalisander — APPROVED** [review]
Good job!

File: benches/src/erc721.rs (hunk 2 above)
Comment (alexfertel) [reply benches/src/erc721.rs:47]: Yeah, this sounds better

**Pull request merged**